circuitpython_build_tools/scripts/build_bundles.py

```diff
@@ -39,7 +39,7 @@
 
 import pkg_resources
 
-NOT_BUNDLE_LIBRARIES = [
+LINUX_LIBRARIES = [
```

Comment:
These are not necessarily Linux only. `adafruit-blinka-bleio` works on Windows and MacOS too. Maybe call these `BLINKA_LIBRARIES` or `HOST_ONLY_LIBRARIES`? I'm not sure.

Reply:
Ok, I like BLINKA_LIBRARIES.